gbif/gbif_name_match/gbif_species_name_match.R

```diff
@@ -47,6 +42,12 @@ request_species_information <- function(df, name_col,
                                                        'status',
                                                        'family')){
 
+    API_terms = c('usageKey', 'scientificName', 'canonicalName', 'rank',
```

Comment:
me nitpicking: use `<-` for assignment. not `=`

Reply:
no worries, you're absolutely right